src/core/components/Root/index.js

```diff
@@ -23,7 +25,11 @@ const Root = ({ children, history, i18n, store, cookies = null }: Props) => (
       <ConnectedRouter history={history}>
         <CookiesProvider cookies={cookies}>
           {/* $FLOW_FIXME: https://github.com/facebook/react/issues/12553 */}
-          <React.StrictMode>{children}</React.StrictMode>
+          {config.get('enableStrictMode') ? (
+            <React.StrictMode>{children}</React.StrictMode>
+          ) : (
+            { children }
```

Comment:
Can you remove the spaces around `children`?

Reply:
I tried but ESLint is complaining:

<img width="628" alt="screen shot 2018-11-23 at 1 38 06 pm" src="https://user-images.githubusercontent.com/13009507/48957192-21851b80-ef25-11e8-98b3-17ac1f060bfd.png">

Reply:
weird.

Reply:
Maybe you could put everything on a single line?

```js
{_config.get('enableStrictMode') ? <React.StrictMode>{children}</React.StrictMode> : children}
```